app/views/stories/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<section class="banner">
+  <div class="container">
+    <div class="content-column banner-content">
+      <h1>Share Your Ideas</h1>
+    </div>
+    <%= image_tag("mapc_logo.svg", class: "logo", alt: t('logo_alt')) %>
+  </div>
+</section>
```

Comment:
Can we create layouts/partials for `.content` and `.banner` sections? It's a lot of code duplication for something that _must_ remain identical across pages.